client/src/components/AppFooter.vue

```diff
@@ -37,10 +37,10 @@
               <i class="fab fa-facebook-square"></i>
```

Comment:
Icons not displaying properly on mine
![image](https://github.com/tomstrong64/MyJourneyFootprint-COM617/assets/128226866/af0cea35-a4d4-43a9-9a21-190460244733)